helm/m3db-operator/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+name: m3db-operator
+version: 0.0.1
+# TODO(PS) - helm has issues with GKE's SemVer
+# Error: Chart requires kubernetesVersion: >=1.10.6 which is incompatible with Kubernetes v1.10.7-gke.2
+#
+#kubeVersion: ">=1.10.7"
+description: m3db-operator provides an automated control plane for managing m3 clusters
```

Comment:
Can we just say "managing m3db clusters" for now and once we manage full instances of the M3 stack we can change it accordingly?

Reply:
sure fair given the progress